deployment/docker-compose/pretix/pretix/etc/pretix.cfg

```diff
@@ -0,0 +1,28 @@
+[pretix]
+instance_name=My pretix installation
+url=http://127.0.0.1/
+currency=EUR
+datadir=/data
+
```

Comment:
The pretix.cfg should contain:
```ini
[pretix]
registration=False
long_sessions=True
```
Or everyone can create a account in the backend, which may not be desirable.